src/entitysdk/token_manager.py

```diff
@@ -0,0 +1,29 @@
+"""Token manager module."""
+
+import os
+from typing import Protocol
+
+from entitysdk.exception import EntitySDKError
+
+
+class TokenManager(Protocol):
+    """Protocol for token managers."""
+
+    def get_token(self) -> str:
+        """Get the token."""
+        ...
```

Comment:
I didn`...` shouldn't be needed if there is a docstring, is it?

Reply:
yeah, removed it.